systemtest/src/test/java/io/strimzi/systemtest/rollingupdate/ManualRollingUpdateST.java

```diff
@@ -144,6 +145,57 @@ void testManualTriggeringRollingUpdate() {
         assertThat(received, is(sent));
     }
 
+    /**
+     * Scenario of the test:
+     * Firstly customer tries to send messages and checks if he is able to consume the messages,
+     * then he delete the cluster CA key. Now should Kafka and Zookeper pods roll.
+     * After the rolling update is done, the CA key/certificate should be renewed,
+     * but should be different than the previous one. Then, again, he tries to
+     * send messages with new generated cluster certificate.
+     */
```

Comment:
I would prefer more structural comment like:

1) Setup Kafka persistent with 3 replicas
2) Create KafkaUser
3) Run producer and consumer to see if cluster is working
4) Remove cluster CA key
5) Kafka and ZK pods should roll, wiat until rolling update finish
6) Check that CA certificates were renewed
7) Try consumer, producer and consume rmeessages again with new certificates

Reply:
100% agree with Frawless here, the scenario should be brief, precise and not meander.

E.g.:
```
Context of the test in a sentence
1. Step 1
2. Step 2
...
```
This means that as the operator evolves if the mechanisms change the disconnect between the comments and the test code are easier to notice and correct or indeed to notice that he operator is not upholding the behaviour our tests are claiming it should have.